public/app/ui/components/display/hierarchy-list.html

```diff
@@ -1,15 +1,17 @@
-<!-- ko if: hasValue -->
-<ul>
-    <!-- ko foreach: { data: value, as: 'item' } -->
-    <li>
-        <!-- ko foreach: { data: item, as: 'hierarchyNode' } -->
-        <!-- ko if: $index() > 0 -->&rArr;<!-- /ko -->
-        <span data-bind="html: hierarchyNode"></span>
+<div class="display display-hierarchy-list">
```

Comment:
Adding these wrapper elements to each of these, it seemed weird having an HTML file that didn't have a single root element.